pwa/src/hooks/useApi.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+const useApi = (route: string) => {
+  const [response, setResponse] = React.useState(null);
```

Comment:
@bmitchinson This is the react state-hook correct? If so, does this eliminate the need for Redux?

Reply:
Great question, a Few things:
- useApi is a hook that acts as a helper function for a component to populate it's component level state with data received from the API.
- The purpose of solutions like "Redux" are to act as a "central state store" to share states between components. useApi doesn't do anything outside of the container it's used in.
- Redux would be pretty big for this application, especially since I don't foresee much being in our central store. (central store -> data shared throughout the application aka across multiple components) I've been loving "easy-peasy" recently as a replacement, but I think the only stuff we'd put in there would be UI navigation items to make form submission simpler.

Reply:
Thank you for explaining this!

Reply:
Of course. They'll be more documentation when it's really put into use later on.